lib/galaxy/tool_util/deps/mulled/invfile.lua

```diff
@@ -42,7 +42,7 @@ end
 
 local conda_image = VAR.CONDA_IMAGE
 if conda_image == '' then
-    conda_image = 'quay.io/condaforge/mambaforge:latest'
+    conda_image = 'quay.io/condaforge/miniforge3:latest'
```

Comment:
We should do this one anyway: https://github.com/conda-forge/miniforge#should-i-choose-one-or-another-going-forward-at-the-risk-of-one-of-them-getting-deprecated

Reply:
Should I take this out and create a separate PR? If so against which release?

Reply:
https://github.com/galaxyproject/galaxy/pull/19327